dist/platform.h

```diff
@@ -44,10 +43,7 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 /*----------------------------------------------------------------------------
 *----------------------------------------------------------------------------*/
-#define INLINE static inline
-
 #define SOFTFLOAT_FAST_INT64
 #define SOFTFLOAT_ROUND_ODD
-#define INLINE_LEVEL 5
```

Comment:
Why removal of this define?

Reply:
It had conflicting symbols when you inline everything, or was missing prototypes.  But when you set the inline level to 0, it doesn't have any problem because it uses the primitives.h definitions.

Reply:
Which symbols conflict? There is actually a point in removal of inline as the module is pretty fat. I would like to make such changes with a specific goal other than workarounding warnings.

I think that the inlined version is too fat so maybe non-inlined variation is small enough and as we do not care about performance too much. But still I would like to see at least a table with:

 - inline mode
 - size of the module
 - benchmark / performance